datumaro/components/progress_reporting.py

```diff
@@ -146,9 +146,15 @@ def start(self, total: int, *, desc: Optional[str] = None):
 
     def report_status(self, progress: int):
         status = str(self._desc) if self._desc else ""
-        status += f"{progress}/{self._total} ({progress/self._total*100:.2f}%)"
+        status += (
+            f" {progress:0{len(str(self._total))}d}/{self._total}"
+            f" ({progress/self._total*100: 6.2f}%)"
```

Comment:
Why is it `6.2f` than `.2f`?

Reply:
To align whole number part as well. It just makes thing a bit prettier like below.
```
(  1.00%)
( 10.00%)
(100.00%)
```